Tests/configure_and_test_integration_instances.py

```diff
@@ -1196,6 +1227,27 @@ def test_pack_zip(content_path, target):
                 zip_file.writestr(test_target, test_file.read())
 
 
+def get_non_added_packs_ids(build: Build):
+    """
+
+    :param build: the build object
+    :return: all non added packs i.e. unchanged packs (dependencies) and modified packs
+    """
+    all_files_changed = run_command(f'git diff --name-status origin/master..refs/heads/{build.branch_name}').split('\n')
+    added_files = filter(lambda x: x.lower().startswith('a') and x.endswith('pack_metadata.json'), all_files_changed)
```

Comment:
you can use this command instead:
`git diff --name-only --diff-filter=M origin/master..refs/heads/{build.branch_name} -- Packs/*/pack_metadata.json`